combine requires clean?

Running this: `./Tools/apache-ant-1.8.2/bin/ant combine runServer`

Does this:

```
Buildfile: /Users/pcozzi/source/cesium/build.xml

build:

setNodePathValue:

setNodePathLocation:

setNodePath:

combineJavaScript.createUnminified:

combineJavaScript.create:
    [mkdir] Created dir: /Users/pcozzi/source/cesium/Build/combineOutput

combineJavaScript.combineCesium.check:

BUILD FAILED
/Users/pcozzi/source/cesium/build.xml:422: The following error occurred while executing this line:
/Users/pcozzi/source/cesium/build.xml:321: The following error occurred while executing this line:
/Users/pcozzi/source/cesium/build.xml:410: cesiumJsFiles doesn't denote a fileset

Total time: 1 second
```

But this works: `./Tools/apache-ant-1.8.2/bin/ant clean combine runServer`

--- a/build.xml
+++ b/build.xml
@@ -408,7 +408,7 @@
 
 	<target name="combineJavaScript.combineCesium.check">
 		<uptodate property="no.combineCesium.create" targetfile="${outputDirectory}/Cesium.js">
-			<srcfiles refid="cesiumJsFiles" />
+			<srcresources refid="cesiumJsFiles" />
 		</uptodate>
 	</target>
 

